server/controllers/article.js

```diff
@@ -58,11 +75,17 @@ export const getArticles = async (req, res) => {
 
 export const getArticle = async (req, res) => {
   try {
-    const { id } = req.params;
+    const article = await findArticle(req);
 
-    const article = await Article.findOne({
-      where: { id }
-    });
+
+    if (article) {
+      const { title, body } = article;
+      const articleReadingTime = getReadingTime(title.concat(' ', body));
```

Comment:
Well done Tosin..Good job. But I feel you should add this function to the before create of article model, such that reading time is determined at the point of creating of an article, thus it can be called generally.

Reply:
Good, however, what happened when a user updates the article? I think it would be nice to calculate only when the article is called.